pages/contribute/README.md

```diff
@@ -36,13 +36,13 @@ We organise frequent gatherings (we call them "hackdays") to discuss the state o
 
 You can also contribute by spreading awareness about the topic to your fellow developers.
 
-* **Share the guide on Twitter** and other channels if you find it useful and want to give something back. We're also available on Twitter under [@A11yDevGuide (Twitter.com)] and on [Facebook].
-* **Talk about accessibility at your local meetup**. Feel free to use the guide as resource. We appreciate every developer who is concerned about accessibility. Let us know so we can share your event.
+- **Share the guide on Twitter** and other channels if you find it useful and want to give something back. We're also available on Twitter under [@A11yDevGuide (Twitter.com)] and on [Facebook].
+- **Talk about accessibility at your local meetup**. Feel free to use the guide as resource. We appreciate every developer who is concerned about accessibility. Let us know so we can share your event.
 
-[Join our Slack team]: https://join.slack.com/t/a11y-dev-guide/shared_invite/enQtMzMwOTkxNTI3NDYwLWI3MjRmZDQwOGMxOGVhNTI2NDg0ODhiMDUyZTQ4MDE4MzU1NmZiMTY2YmNjNjliYWEzODhjZDYwYjA1MDU4NmU
-[create an issue on GitHub]: https://github.com/Access4all/adg/issues
-[Access4all/adg (GitHub.com)]: https://github.com/Access4all/adg
-[Access4all/adg/CONTRIBUTING.md (GitHub.com)]: https://github.com/Access4all/adg/blob/master/CONTRIBUTING.md 
-[Join our Meetup.com group]: https://www.meetup.com/Accessibility-Developer-Guide-ADG/
-[@A11yDevGuide (Twitter.com)]: https://twitter.com/A11yDevGuide
-[Facebook]: https://www.facebook.com/AccessibilityDeveloperGuide
+[join our slack team]: https://join.slack.com/t/a11y-dev-guide/shared_invite/zt-481zt544-HZCboLee6JL__6LnHl1N5w
```

Comment:
Thanks for the new link!

What is the reason for lowercasing some words below?

Reply:
@gurix honestly, no idea about the case-change. That was probably done by the formatter based on the fact those "sentences" are used in that exact case in the text of the markdown file. I then decided not to work against the formatter and instead tested if it makes a technical difference. And It seems that it doesn't.

Reply:
@renestalder, IMHO the previous version was more readable. Not completely consistent as the `create` should probably have been capitalized, but still easier to parse.

Do you know what formatting tool is responsible for this change?

Reply:
@backflip looks like that's Prettier activated in my vscode. I can also restore the formatting in the documents and probably should deactivate the prettier extension in that project.

Just for our both understanding: That section of the markdown file isn't rendered/not visible in the HTML output. These are just references to be used in the text. If you look at the [preview](https://deploy-preview-343--accessibility-developer-guide.netlify.app/contribute/) the case in the text that calls those footer references uses whatever casing the text defines. So the casing doesn't really matter here.

Reply:
Wow, thanks for pointing this out, I did not know about this markdown feature. Should we just move the links into the text as this would IMHO be way more maintainable?

Reply:
@backflip hmm, I mean it took me under 10 seconds to replace the link and I could be sure It was replaced in the whole text where otherwise I would have had to read through the text to ensure I replace all references. The good part of this markdown feature is that you can reuse the same link multiple times and only have to maintain it in one place. IMHO that makes it the more maintainable solution than inline links in longer texts.

It doesn't really matter to me. Your call. I initially implemented this page with this markdown feature for exactly the reason to be able to easily replace references. If others find it more confusing, then yes, it makes sense to use inline links instead.

(in addition, assuming we might use a CMS at some point, that markdown feature will anyway no longer work as most CMS don't implement it)

Reply:
@renestalder Fair enough, I was only thinking about the case where I would want to edit the link text (which I would have to edit twice unless I'm mistaken). I did not think about your case which is definitely more efficient.